helm/botkube/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: botkube
 home: https://botkube.io
-version: v9.99.9-dev
-appVersion: v9.99.9-dev
+version: v0.15.0-rc.1
+appVersion: v0.15.0-rc.1
```

Comment:
I think all the versions shouldn't change as the release pipeline will override all of the stuff. Correct?

Reply:
they will be overwritten once release is finalized